sherpa/astro/ui/tests/test_astro_ui_plot.py

```diff
@@ -326,8 +330,65 @@ def test_get_arf_plot(idval, clean_astro_ui):
 
     setup_example(idval)
     if idval is None:
+        ui.set_analysis("energy")
         ap = ui.get_arf_plot()
     else:
+        ui.set_analysis(idval, "energy")
+        ap = ui.get_arf_plot(idval)
+
+    assert isinstance(ap, sherpa.astro.plot.ARFPlot)
+
+    assert ap.xlo == pytest.approx(_energies_lo)
+    assert ap.xhi == pytest.approx(_energies_hi)
+
+    assert ap.y == pytest.approx(_arf)
+
+    assert ap.title == 'test-arf'
+    assert ap.xlabel == 'Energy (keV)'
+
+    # the y label depends on the backend (due to LaTeX)
+    # assert ap.ylabel == 'cm$^2$'
```

Comment:
Not complete, but better than skipping.
```suggestion
    assert 'cm' in ap.ylabel
    assert '2' in ap.ylabel
```